lib/mlir/Dialect/Clift/IR/CliftOps.cpp

```diff
@@ -234,65 +235,76 @@ mlir::LogicalResult clift::ModuleOp::verify() {
   return Validator.visitOp(*this);
 }
 
-//===-----------------------------------------------------------------========//
-// Code for clift::AssignLabelOp.
-//===----------------------------------------------------------------------===//
+//===----------------------------- Statements -----------------------------===//
 
-mlir::LogicalResult
-mlir::clift::AssignLabelOp::canonicalize(mlir::clift::AssignLabelOp Op,
-                                         mlir::PatternRewriter &Rewriter) {
-  for (const mlir::OpOperand &Use : Op.getLabel().getUses())
-    if (mlir::isa<mlir::clift::GoToOp>(Use.getOwner()))
-      return mlir::success();
+//===----------------------------- MakeLabelOp ----------------------------===//
 
-  Rewriter.eraseOp(Op);
-  return mlir::success();
+static std::pair<size_t, size_t> getMakeLabelOpUsers(MakeLabelOp Op) {
```

Comment:
Rename the function to `getNumLabelUsers` or `countLabelUsers` please (`getNum` is ugly but it's typical in LLVM). The current name suggests you're getting users, while you're only counting them.